src/main/scala/spark/dicom/deidentifier/DicomDeidentifier.scala

```diff
@@ -39,11 +39,14 @@ object DicomDeidentifier {
     *   value represenation of the deidElem
     * @param config
     *   configuration settings to be used in de-identification
+    * @param salt
+    *   salt to use in the pseudonumization action
     */
   def getAction(
       deidElem: DicomDeidElem,
       vr: VR,
-      config: Map[DeidOption, Boolean]
+      config: Map[DeidOption, Boolean] = Map.empty,
+      salt: String = scala.util.Random.alphanumeric take 20 mkString
```

Comment:
It's better not to use effectful code as default function arguments value.

Reply:
Makes sense. Would you propose no default value? Or just empty string as default

Reply:
In the lowest levels of the API, I prefer using as little default values as possible, but that a personal preference.